src/main.rs

```diff
@@ -84,6 +92,36 @@ fn custom_maskfile_path_arg<'a, 'b>() -> Arg<'a, 'b> {
     custom_maskfile_path
 }
 
+fn shell_completion_subcommand<'a, 'b>(cli_app: App<'a, 'b>) -> App<'a, 'b> {
+    cli_app.subcommand(
+        SubCommand::with_name("init")
+            .about("Prints the shell function used to execute `mask`")
+            .arg(
+                &Arg::with_name("shell")
+                    .value_name("SHELL")
+                    .help("The name of the currently running shell")
+                    .possible_values(&["bash", "zsh", "fish", "elvish", "powershell"])
```

Comment:
Just curious, are these all the possible values clap supports?

Reply:
yups
https://docs.rs/clap/2.33.0/clap/enum.Shell.html